packages/components/src/range-control/index.js

```diff
@@ -77,6 +77,13 @@ function RangeControl(
 		initial: initialPosition,
 	} );
 	const isResetPendent = useRef( false );
+
+	const isStepAny = step === 'any';
+
+	if ( isStepAny ) {
+		showTooltipProp = withInputField = false;
```

Comment:
Nitpick: I think this would be better split into two lines. Not sure it's official in the code style guidelines but it would make this consistent with the rest of the codebase.

```suggestion
		showTooltipProp = false;
		withInputField = false;
```

Reply:
Thanks for the note on this. I'd been curious to see if it lints but might have forgotten about it.